src/components/Info/Index.js

```diff
@@ -1,19 +1,47 @@
+import {
+  InfoContainerCustom,
+  TextContainer,
+  TextInfoBox,
+  DescriptionText,
+  TextEspacement,
+  AbilitiesContainer,
+  InfoRequestCustom,
+} from "./styled";
+
 export default function Info({
   pokemonInfo,
   setPokemonInfo,
   pokemonDescription,
   setPokemonDescription,
 }) {
   return (
-    <div>
-      <p>{pokemonInfo.weight}</p>
-      <p>{pokemonInfo.height}</p>
-      <p>{pokemonInfo.abilities && pokemonInfo.abilities[0].ability.name}</p>
-      <p>{pokemonInfo.abilities && pokemonInfo.abilities[1].ability.name}</p>
-      <p>
+    <InfoContainerCustom>
+      <TextContainer>
+        <TextInfoBox>
+          <TextEspacement>Weight</TextEspacement>
+          <InfoRequestCustom>{pokemonInfo.weight} kg</InfoRequestCustom>
+        </TextInfoBox>
+        <TextInfoBox>
+          <TextEspacement>Height</TextEspacement>
+          <InfoRequestCustom>{pokemonInfo.height} m</InfoRequestCustom>
+        </TextInfoBox>
+        <TextInfoBox>
+          <TextEspacement>Abilities</TextEspacement>
+          <AbilitiesContainer>
+            <p>
+              {pokemonInfo.abilities && pokemonInfo.abilities[0].ability.name}
```

Comment:
Be careful, you could have an empty array.
ex: pokemon.abilities = []
if you try to access first index (e.g: 0) it will crash
You have to test if pokemonInfo.abilities length